src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/NotifyConfiguration.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import uk.gov.service.notify.NotificationClient
+
+@Configuration
+class NotifyConfiguration(@Value("\${notify.apikey:}") private val apiKey: String) {
```

Comment:
notice the trailing colon on the property here (it means default value of empty string). this is the only way i could get this to work without having the property name in the config file. we only want this value to come from env vars though.